examples/TargetingConsoleApp/appsettings.json

```diff
@@ -24,6 +24,47 @@
           }
         }
       ]
+    },
+    "ShoppingCart": {
+      "Assigner": "Targeting",
```

Comment:
>Targeting [](http://example.com/codeflow?start=19&length=9)

Can "Targeting" and "Microsoft.Targeting" be used interchangeably?

Reply:
Yes. Feature Filter and Assigner names support namespacing. That way somebody can have their own targeting filter.